docs/content/documentation/getting-started/configuration.md

```diff
@@ -130,6 +130,9 @@ skip_anchor_prefixes = [
     "https://caniuse.com/",
 ]
 
+# Check for links to anchors defined in HTML rather than markdown
+heuristic_link_check = true
```

Comment:
in the end since it's only ran on local failures, it's probably fine to not need an option

Reply:
Your call, but it would give the option of disabling something that can produce failures. Switching the default around might be easier?

Reply:
There's already the workaround of creating the link in HTML directly